packages/dds/tree/src/test/shared-tree/fuzzEditGenerator.ts

```diff
@@ -65,6 +64,11 @@ export interface TreeContext {
 	treeIndex: number;
 }
 
+export interface NodeRangePath {
+	firstNodePath: UpPath;
```

Comment:
Nit: I think `firstNode` is sufficient. We generally avoid including information about the type in the name since in most cases it's easy to lookup the type.